jerry-core/rcs/rcs-records.c

```diff
@@ -222,7 +222,7 @@ rcs_record_get_field (rcs_record_t *rec_p, /**< record */
   JERRY_ASSERT (field_pos + field_width <= RCS_DYN_STORAGE_LENGTH_UNIT * JERRY_BITSINBYTE);
 
   uint32_t value = *(uint32_t *) rec_p;
-  return (uint32_t) jrt_extract_bit_field (value, field_pos, field_width);
+  return JRT_EXTRACT_BIT_FIELD (uint32_t, value, field_pos, field_width);
```

Comment:
Sorry, I have to revert that. It cannot be done. There is something nasty going on around `rcs_record_t`. It is declared as `uint8_t` but according to rcs-records.h, it is always at least 32 bits (header) and then comes some payload of "arbitrary" length. That must be why it is type cast to `uint32_t`.